modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/PrecisionAtKTests.java

```diff
@@ -58,7 +58,7 @@ public void testPrecisionAtFiveCalculation() {
         assertEquals(1, ((PrecisionAtK.Detail) evaluated.getMetricDetails()).getRetrieved());
     }
 
-    public void testPrecisionAtFiveIgnoreOneResult() {
+    public void testPrecisionAtKIgnoreOneResult() {
```

Comment:
Yeah, I was trying to fix the test names in `PrecisionAtK` which I found contradictory between the test/method name and the details of the test. I added the `AtFive` when the test was specifically not using the default `k` since it better describes the test case. I'm happy to drop all those though and simplify the names.